EIPS/eip-1024.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 1024
+title: Add web3.eth.encrypt and web3.eth.decrypt functions
+author: Tope Alabi <alabitemitope@gmail.com>
+status: Draft
+type: Interface Track
+created: 2018-05-14
+---
+
+
+### Abstract
```

Comment:
All these sections seem to be indented one level too deep, it should be `## Abstract`.